src/configurator/__init__.py

```diff
@@ -1 +1 @@
-from .configurator import configure, parse_config
+from .configurator import create_build, parse_config
```

Comment:
Place elements in `__all__` to explicitly show they are public API
I think `create_build` is private function, that is used inside the configurator only, isn't it?

```suggestion
from .configurator import parse_config

__all__ = ["parse_config"]
```